src/aleph/http/client_middleware.clj

```diff
@@ -131,9 +131,9 @@
     (dissoc m k)))
 
 (defn url-encode
-  ([s]
+  ([^String s]
```

Comment:
As far as I know this type hint is not needed, only the one at line 136 is required.

Reply:
@rborer Yeah, you're right. Just a habit to ensure consistency in case if the first clause should be reimplemented in future (to keep user-facing API stable).

Reply:
@kachayev Oh, good point about keeping the API stable :+1: .

Reply:
@rborer 👍